rclcpp/include/rclcpp/executors/static_executor_entities_collector.hpp

```diff
@@ -72,9 +72,15 @@ class StaticExecutorEntitiesCollector final
   void
   fini();
 
+  /// Execute the waitable.
   RCLCPP_PUBLIC
   void
-  execute() override;
+  execute(std::shared_ptr<void> & data) override;
+
+  /// Prepare the waitable for execution.
```

Comment:
I think the documentation here could be a bit better. Prepare it how?

Reply:
Fixed in https://github.com/ros2/rclcpp/pull/1241/commits/4d21b6eb6404b32b687303915c9c5b36e73812e9.